wp-content/plugins/core/src/Settings/Theme_Options.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Settings;
+
+use Tribe\Libs\ACF\ACF_Settings;
+
+class Theme_Options extends ACF_Settings {
+
+	public const SLUG = 'theme-options';
+
+	public function get_title(): string {
+		$current_theme = wp_get_theme();
+		$theme_name    = $current_theme->exists() ? $current_theme->get( 'Name' ) : 'Theme';
```

Comment:
I wonder if we should just simplify this to not use the theme name and just standardize is across all projects to `Settings` or `Theme Options` or `Theme Settings`. 

That way, if we need to write documentation, help developers or clients find this page, it's always named the same thing.

This also points out a grammar error in our stub theme name, should be `Your Client's Name` :smile:

Reply:
Yeah, Chris & I discussed that a bit. I'd prefer to use the actual theme name for the label to make it clear it's not a generic page and instead specific to the project/client. The same issue we have with "General Settings" would apply to "Theme Options" imho.